3party/opening_hours/opening_hours_tests/opening_hours_tests.cpp

```diff
@@ -793,6 +793,11 @@ BOOST_AUTO_TEST_CASE(OpeningHoursWeekdays_TestParseUnparse)
     auto const parsedUnparsed = ParseAndUnparse<osmoh::Weekdays>(rule);
     BOOST_CHECK_EQUAL(parsedUnparsed, rule);
   }
+  {
+    auto const rule = "Sa";
+    auto const parsedUnparsed = ParseAndUnparse<osmoh::Weekdays>(rule);
```

Comment:
https://en.wiktionary.org/wiki/weekday <https://en.wiktionary.org/wiki/weekday>

> On Apr 1, 2016, at 11:21, mgsergio <notifications@github.com> wrote:
> 
> In 3party/opening_hours/opening_hours_tests/opening_hours_tests.cpp <https://github.com/mapsme/omim/pull/2655#discussion_r58175414>:
> 
> > @@ -793,6 +793,11 @@ BOOST_AUTO_TEST_CASE(OpeningHoursWeekdays_TestParseUnparse)
> >      auto const parsedUnparsed = ParseAndUnparse<osmoh::Weekdays>(rule);
> >      BOOST_CHECK_EQUAL(parsedUnparsed, rule);
> >    }
> > +  {
> > +    auto const rule = "Sa";
> > +    auto const parsedUnparsed = ParseAndUnparse<osmoh::Weekdays>(rule);
> В осме называется wday http://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#wday <http://wiki.openstreetmap.org/wiki/Key:opening_hours/specification#wday>
> Думаю, можно оставить так ибо opening_hours это реализация осмовской спецификации.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2655/files/4a4de47aae7310c4ae4d1aca101b3efc68e74915#r58175414>